operators/constellation-node-operator/controllers/scalinggroup_controller.go

```diff
@@ -92,8 +93,10 @@ func (r *ScalingGroupReconciler) Reconcile(ctx context.Context, req ctrl.Request
 	}
 	meta.SetStatusCondition(&desiredScalingGroup.Status.Conditions, outdatedCondition)
 	if err := r.Status().Update(ctx, &desiredScalingGroup); err != nil {
-		logr.Error(err, "Unable to update AutoscalingStrategy status")
-		return ctrl.Result{}, err
+		if !k8sErrors.IsConflict(err) {
+			logr.Error(err, "Unable to update Scaling Group status")
+		}
+		return ctrl.Result{Requeue: true}, err
```

Comment:
Why only log the error in case its a conflict error?
Can we safely requeue the request in both cases?

Reply:
> I disabled the logging in case of a conflict to reduce the number of operator logs a bit. But I hove no strong opinion here.

Makes sense, lets keep it disabled.

I made this annotation since we didn't set the `Requeue` option before, but are now setting it for both conflict and other errors.
I guess this could speed up the upgrade process a bit?

Reply:
Ah sorry, I missed that I also set the re-queue now. This is because I saw a conflict in that line and therefore almost went for the retryOnConflict versioin, but since it should be similar to just re-queuing, I went with that solution.